client/modules/sample/components/Header/__tests__/Header-spec.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import expectReactShallow from 'expect-react-shallow'
+
+import Header from '..'
+import shallowRender from 'utils/shallowRender'
+
+describe('Header', () => {
+  function className(component) {
+    return shallowRender(component).props.className
+  }
+
+  test('renders the root element as a div by default', () => {
+    expectReactShallow(<Header />)
+      .to.have.rendered(<h1 />)
+  })
+
+  test('renders header with the "heading" class', () => {
+    expect(className(<Header />)).toContain('heading')
```

Comment:
Would it be safer to import `styles.scss` here and use `styles.heading` here?  That way, we're not coupled to the mechanism we're using to fake out CSS modules imports.